src/main/java/io/vertx/proton/ProtonClientOptions.java

```diff
@@ -314,6 +324,48 @@ public ProtonClientOptions setLocalAddress(String localAddress) {
   }
 
   /**
+   * Override the host to use in the connection open frame. If this is not set, the hostname specified in
```

Comment:
Should mention it explicitly sets the SNI server name too, but that can be overridden using the SniServerName option. Perhaps update getter description accordingly.